Maya 2020 picked wrong deleteNode method in translatorCurves

**Describe the issue**
In the code below, Maya version is used in condition for selecting correct `MDGModifier::deleteNode`
https://github.com/Autodesk/maya-usd/blob/82ac60a9c2b74302e1513a91a6fcb258bde4a894/lib/mayaUsd/fileio/translators/translatorCurves.cpp#L64-L68

But the method `MStatus MDGModifier::deleteNode( const MObject & node, bool includeParents )` was introduced in `Maya 2020.3`, so simply using Maya App version as condition will not work for Maya 2020-2020.2.

**Build log**
```log
..\maya-usd-0.14.0\lib\mayaUsd\fileio\translators\translatorCurves.cpp(65): error C2660: 'Autodesk::Maya::OpenMaya202 00000::MDGModifier::deleteNode': function does not take 2 arguments [..\_usd_maya\lib\mayaUsd\mayaUsd.vcxproj]
  C:\Program Files\Autodesk\Maya2020\include\maya/MDGModifier.h(114): note: see declaration of 'Autodesk::Maya::OpenMaya20200000::MDGModifier::deleteNode'
```

<details>

<summary>Full build log</summary>

```
rezup (dev/local) davidlatwe.lai@MOONSHINEEXP27 C:\Users\davidlatwe.lai
$ rez-deliver usd_maya-0.14.0
Mode: install (-> C:\Users\davidlatwe.lai\rez\packages\install)

Following packages will be deployed:
----------------------------------------------------------------------
 python-2.7-venv        | (external)
 default-1.0.0-m1       | (external)
 rezutil-1.4.6-m2[0]    | (installed)
 maya-2020-m3[0]        | (installed)
 platform-windows       | (installed)
 vcvars64-2017-m1[0]    | (installed)
 cmake-3.18.2-m1[0]     | (installed)
 arch-AMD64             | (installed)
 os-windows-10.0        | (installed)
 rez-2.100.0[0]         | (installed)
 python-3.7-venv        | (external)
 rez-2.100.0[1]         | (installed)
 pipz-1.3.0.dev.0[0]    | (external)
 PyOpenGL-3.1.5-m1[0]   | (external)
 PySide-1.2.4-m1[0]     | (external)
 Qt.py-1.3.1-m2[2]      | (external)
 doxygen-1.8.20-m1[0]   | (installed)
 graphviz-2.44.1-m1[0]  | (installed)
 nasm-2.14.02-m1[0]     | (installed)
 jinja2-2.11.2-m1[0]    | (installed)
 usd-21.08-m2[0]        | (installed)
 maya_devkit-2020-m1[0] | (installed)
 qt-5.12.5-m1[0]        | (installed)
 future-0.18.2-m1[0]    | (installed)
 usd_maya-0.14.0-m2[0]  | (ready)
Do you want to continue ? [Y/n]
y
Running command:
    ['C:\\Users\\davidlatwe.lai\\.rezup\\dev\\revisions\\1637232169.1712413\\venv\\rez\\Scripts\\python.exe', '-m', 'deliver.install', 'usd_maya-0.14.0-m2[0]', '--install', '--variants', '0']



================================================================================
Building usd_maya-0.14.0-m2...
================================================================================

================================================================================
Building variant 0 (1/1)...
================================================================================
Resolving build environment: python maya-2020+ platform-windows maya-2020 python-2.7 rezutil-1+ vcvars64-2017 cmake-3.14+ usd-21.08 maya_devkit-2020 qt-5 jinja2 future
resolved by davidlatwe.lai@MoonShineExp27, on Tue Nov 23 18:44:14 2021, using Rez v2.98.0+m

requested packages:
python
maya-2020+
platform-windows
maya-2020
python-2.7
rezutil-1+
vcvars64-2017
cmake-3.14+
usd-21.08
maya_devkit-2020
qt-5
jinja2
future
~platform==windows  (implicit)
~arch==AMD64        (implicit)
~os==windows-10.0   (implicit)

resolved packages:
PyOpenGL-3.1.5-m1    t:\rez-studio\packages\release\PyOpenGL\3.1.5-m1\python-2.7
PySide-1.2.4-m1      t:\rez-studio\packages\release\PySide\1.2.4-m1\os-windows\python-2.7
Qt.py-1.3.1-m2       t:\rez-studio\packages\release\Qt.py\1.3.1-m2\PySide
arch-AMD64           t:\rez-studio\packages\release\arch\AMD64
cmake-3.18.2-m1      t:\rez-studio\packages\release\cmake\3.18.2-m1\default
default-1.0.0-m1     t:\rez-studio\packages\release\default\1.0.0-m1
future-0.18.2-m1     c:\users\davidlatwe.lai\rez\packages\install\future\0.18.2-m1\python-2.7           (local)
jinja2-2.11.2-m1     t:\rez-studio\packages\release\jinja2\2.11.2-m1\python-2.7
maya-2020-m3         c:\users\davidlatwe.lai\rez\packages\install\maya\2020-m3\default                  (local)
maya_devkit-2020-m1  c:\users\davidlatwe.lai\rez\packages\install\maya_devkit\2020-m1\platform-windows  (local)
os-windows-10.0      t:\rez-studio\packages\release\os\windows-10.0
platform-windows     t:\rez-studio\packages\release\platform\windows
python-2.7-venv      t:\rez-studio\packages\combined\python.py<2.7-venv>[]
qt-5.12.5-m1         c:\users\davidlatwe.lai\rez\packages\install\qt\5.12.5-m1\default                  (local)
rezutil-1.4.6-m2     t:\rez-studio\packages\release\rezutil\1.4.6-m2\default
usd-21.08-m2         t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7
vcvars64-2017-m1     t:\rez-studio\packages\release\vcvars64\2017-m1\default

Invoking custom build system...
Running build command: python c:\users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0/rezbuild.py install
Collect visual studio developer cmd env vars..
Initialize visual studio developer cmd env..
File exists, skip download: c:\users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\v0.14.0.zip
Extracted flag found, skip extract. Remove flag file to re-extract if needed:
        c:\users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\extract_dir\EXTRACTED
-- Selecting Windows SDK version 10.0.17763.0 to target Windows 10.0.18363.
-- _PYTHON_VERSION_LIST: 2.7.18.final.0
-- PYTHON_PREFIX: C:\python27_64\rez.venv
-- PYTHON_INCLUDE_DIR: C:\python27_64\rez.venv\include
-- PYTHON_SITE_PACKAGES: C:\python27_64\rez.venv\Lib\site-packages
-- PYTHON_MODULE_EXTENSION: .pyd
-- PYTHON_IS_DEBUG: 0
-- PYTHON_SIZEOF_VOID_P: 8
-- PYTHON_LIBRARY_SUFFIX: 27
-- PYTHON_LIBDIR:
-- PYTHON_MULTIARCH:
-- Build MayaUSD with Python3 = OFF
--    PYTHON_INCLUDE_DIRS = C:/python27_64/rez.venv/include
--    PYTHON_LIBRARIES    = C:/python27_64/rez.venv/libs/Python27.lib
--    Python_EXECUTABLE   = C:/python27_64/rez.venv/Scripts/python.exe
-- Using Maya Light API Version 1
-- USD include dir: T:/rez-studio/packages/release/usd/21.08-m2/platform-windows/python-2.7/payload/include
-- USD library dir: T:/rez-studio/packages/release/usd/21.08-m2/platform-windows/python-2.7/payload/lib
-- USD version: 0.21.8
-- USD Boost::boost version: 107000
Updated PYTHONPATH: c:\users\davidlatwe.lai\rez\packages\install\future\0.18.2-m1\python-2.7/python;t:\rez-studio\packages\release\jinja2\2.11.2-m1\python-2.7/python;t:\rez-studio\packages\release\Qt.py\1.3.1-m2\PySide/python;t:\rez-studio\packages\release\PySide\1.2.4-m1\os-windows\python-2.7/python;t:\rez-studio\packages\release\PyOpenGL\3.1.5-m1\python-2.7/python;t:\rez-studio\packages\release\rezutil\1.4.6-m2\default/python;t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7/payload/lib/python;T:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\lib\python
Updated PATH: c:\users\davidlatwe.lai\rez\packages\install\future\0.18.2-m1\python-2.7/bin;t:\rez-studio\packages\release\jinja2\2.11.2-m1\python-2.7/bin;t:\rez-studio\packages\release\Qt.py\1.3.1-m2\PySide/bin;t:\rez-studio\packages\release\PySide\1.2.4-m1\os-windows\python-2.7/bin;t:\rez-studio\packages\release\PyOpenGL\3.1.5-m1\python-2.7/bin;C:/Program Files/CMake/bin;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCPackages;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\bin\Roslyn;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Team Tools\Performance Tools\x64;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Team Tools\Performance Tools;C:\Program Files (x86)\Microsoft Visual Studio\Shared\Common\VSPerfCollectionTools\\x64;C:\Program Files (x86)\Microsoft Visual Studio\Shared\Common\VSPerfCollectionTools\;C:\Program Files (x86)\Windows Kits\10\bin\10.0.17763.0\x64;C:\Program Files (x86)\Windows Kits\10\bin\x64;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\\MSBuild\15.0\bin;C:\Windows\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin;C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja;C:\python27_64\rez.venv\Scripts;C:/Program Files/Autodesk/Maya2020/bin;C:/Program Files/Common Files/Autodesk Shared/;C:/Program Files (x86)/Autodesk/Backburner/;t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7/payload/bin;t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7/payload/plugin/usd;t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7/payload/lib;C:/Qt/Qt5.12.5/5.12.5/msvc2017_64/bin;C:/Qt/Qt5.12.5/5.12.5/msvc2017_64/lib;C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\Common Files\Autodesk Shared\;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files (x86)\Autodesk\Backburner\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Python37_64;C:\Python37_64\Scripts;C:\miniconda3;C:\miniconda3\Scripts;C:\Users\davidlatwe.lai\Documents\maya\;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Users\davidlatwe.lai\AppData\Local\Programs\Python\Launcher\;C:\Users\DAVIDL~1.LAI\AppData\Local\Temp\tmpmao26l3n\apps\python\current;C:\Users\DAVIDL~1.LAI\AppData\Local\Temp\tmpmao26l3n\apps\python\current\Scripts;C:\Users\DAVIDL~1.LAI\AppData\Local\Temp\tmpswvow8t_\apps\python37\current\scripts;C:\Users\DAVIDL~1.LAI\AppData\Local\Temp\tmp_vd_ltxi\apps\python37\current\scripts;C:\Users\DAVIDL~1.LAI\AppData\Local\Temp\tmpfrumswbn\apps\python\current;C:\Users\DAVIDL~1.LAI\AppData\Local\Temp\tmpfrumswbn\apps\python\current\Scripts;C:\Users\DAVIDL~1.LAI\AppData\Local\Temp\tmp1c756n7a\apps\python\current;C:\Users\DAVIDL~1.LAI\AppData\Local\Temp\tmp1c756n7a\apps\python\current\Scripts;C:\Users\davidlatwe.lai\AppData\Local\Microsoft\WindowsApps;C:\Program Files\JetBrains\PyCharm Community Edition 2020.1.2\bin;C:\Program Files\mongosh\;C:\act;T:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\bin;T:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\lib
-- UFE include dir: C:/Users/davidlatwe.lai/rez/packages/install/maya_devkit/2020-m1/platform-windows/devkitBase/devkit/ufe/include
-- UFE library: C:/Users/davidlatwe.lai/rez/packages/install/maya_devkit/2020-m1/platform-windows/devkitBase/devkit/ufe/lib/ufe_1.lib
-- UFE version: 1.0.0
-- Building with UFE 1.0.0 features enabled.
-- Building with Qt features enabled.
-- **** Building Gulark From Github Repository.
-- Populating gulark
-- Selecting Windows SDK version 10.0.17763.0 to target Windows 10.0.18363.
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/_deps/gulark-subbuild
Microsoft (R) Build Engine for .NET Framework 15.9.21+g9802d43bc3 版
Copyright (C) Microsoft Corporation. 著作權所有，並保留一切權利。

  Performing update step for 'gulark-populate'
  No patch step for 'gulark-populate'
  No configure step for 'gulark-populate'
  No build step for 'gulark-populate'
  No install step for 'gulark-populate'
  No test step for 'gulark-populate'
  Completed 'gulark-populate'
-- MAYA_HAS_DEFAULT_MATERIAL_API is FALSE
-- MAYA_NEW_POINT_SNAPPING_SUPPORT is FALSE
-- MAYA_HAS_CRASH_DETECTION is FALSE
-- MAYA_CURRENT_UFE_CAMERA_SUPPORT is FALSE
-- MAYA_MRENDERITEM_UFE_IDENTIFIER_SUPPORT is FALSE
-- promoting: C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/maya-usd-0.14.0/lib/mayaUsd/base/mayaUsd.h.src
Processing schema classes:
MayaReference, ALMayaReference
Loading Templates from t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\lib\usd\usd\resources\codegenTemplates
Writing API:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\api.h
Writing Schema Tokens:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\tokens.h
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\tokens.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\wrapTokens.cpp
Generating Classes:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\MayaReference.h
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\MayaReference.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\wrapMayaReference.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\ALMayaReference.h
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\ALMayaReference.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\wrapALMayaReference.cpp
Loading Templates from t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\lib\usd\usd\resources\codegenTemplates
            wrote C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\plugInfo.json
Generating Schematics:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\schemas\generatedSchema.usda
========== Pixar USD Plugin ==========
Compiling C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/plugin/pxr/__init__.py ...
========== AL USD Plugin ==========
CMake Deprecation Warning at plugin/al/CMakeLists.txt:7 (cmake_policy):
  The OLD behavior for policy CMP0026 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.


-- Found Boost 1.70.0 at T:/rez-studio/packages/release/usd/21.08-m2/platform-windows/python-2.7/payload/lib/cmake/Boost-1.70.0
--   Requested configuration: QUIET REQUIRED
-- Found boost_headers 1.70.0 at T:/rez-studio/packages/release/usd/21.08-m2/platform-windows/python-2.7/payload/lib/cmake/boost_headers-1.70.0
-- Found Boost: T:/rez-studio/packages/release/usd/21.08-m2/platform-windows/python-2.7/payload/lib/cmake/Boost-1.70.0/BoostConfig.cmake (found version "1.70.0")
-- Found Boost 1.70.0 at T:/rez-studio/packages/release/usd/21.08-m2/platform-windows/python-2.7/payload/lib/cmake/Boost-1.70.0
--   Requested configuration: QUIET REQUIRED COMPONENTS python
-- Found boost_python 1.70.0 at T:/rez-studio/packages/release/usd/21.08-m2/platform-windows/python-2.7/payload/lib/cmake/boost_python-1.70.0
--   boost_python27-vc141-mt-x64-1_70.lib
-- Adding boost_python dependencies: headers
-- Found Boost: T:/rez-studio/packages/release/usd/21.08-m2/platform-windows/python-2.7/payload/lib/cmake/Boost-1.70.0/BoostConfig.cmake (found version "1.70.0") found components: python
Compiling C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/plugin/al/lib/AL_USDMaya/lib/python/AL/usdmaya/__init__.py ...
-- Building AL_USDMayaSchemas
Processing schema classes:
ALModelAPI, ALFrameRange
Loading Templates from t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\lib\usd\usd\resources\codegenTemplates
Writing API:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\api.h
Writing Schema Tokens:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\tokens.h
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\tokens.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\wrapTokens.cpp
Generating Classes:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\ModelAPI.h
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\ModelAPI.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\wrapModelAPI.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\FrameRange.h
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\FrameRange.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\wrapFrameRange.cpp
Loading Templates from t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\lib\usd\usd\resources\codegenTemplates
            wrote C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\plugInfo.json
Generating Schematics:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\maya\generatedSchema.usda
Compiling C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/AL/__init__.py ...
Compiling C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/AL/usd/__init__.py ...
Compiling C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/AL/usd/schemas/__init__.py ...
Processing schema classes:
ALExamplePolyCubeNode
Loading Templates from t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\lib\usd\usd\resources\codegenTemplates
Writing API:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\api.h
Writing Schema Tokens:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\tokens.h
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\tokens.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\wrapTokens.cpp
Generating Classes:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\ExamplePolyCubeNode.h
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\ExamplePolyCubeNode.cpp
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\wrapExamplePolyCubeNode.cpp
Loading Templates from t:\rez-studio\packages\release\usd\21.08-m2\platform-windows\python-2.7\payload\lib\usd\usd\resources\codegenTemplates
            wrote C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\plugInfo.json
Generating Schematics:
        unchanged C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\plugin\al\schemas\AL\usd\schemas\mayatest\generatedSchema.usda
Compiling C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/AL/__init__.py ...
Compiling C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/AL/usd/__init__.py ...
Compiling C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya/AL/usd/schemas/__init__.py ...
-- Building AL_USDMayaTranslators
========== ADSK USD Plugin ==========
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/davidlatwe.lai/pipeline/rez-kit/production/usd_maya/0.14.0/build/0/_usd_maya
Microsoft (R) Build Engine for .NET Framework 15.9.21+g9802d43bc3 版
Copyright (C) Microsoft Corporation. 著作權所有，並保留一切權利。

  mayaUsdUtils.vcxproj -> C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\usd\utils\Release\mayaUsdUtils.dll
  translatorCurves.cpp
C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\maya-usd-0.14.0\lib\mayaUsd\fileio\translators\translatorCurves.cpp(65): error C2660: 'Autodesk::Maya::OpenMaya202 00000::MDGModifier::deleteNode': function does not take 2 arguments [C:\Users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0\build\0\_usd_maya\lib\mayaUsd\mayaUsd.vcxproj]
  C:\Program Files\Autodesk\Maya2020\include\maya/MDGModifier.h(114): note: see declaration of 'Autodesk::Maya::OpenMaya20200000::MDGModifier::deleteNode'
Traceback (most recent call last):
  File "c:\users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0/rezbuild.py", line 121, in <module>
    targets=sys.argv[1:])
  File "c:\users\davidlatwe.lai\pipeline\rez-kit\production\usd_maya\0.14.0/rezbuild.py", line 104, in build
    lib.run_cmake(source_root, install_path, extra_args=extra_args)
  File "t:\rez-studio\packages\release\rezutil\1.4.6-m2\default\python\rezutil\lib.py", line 233, in run_cmake
    subprocess.check_call(build)
  File "C:\python27_64\lib\subprocess.py", line 190, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmake', '--build', '.', '--config', 'Release', '--target', 'install']' returned non-zero exit status 1
18:45:01 ERROR    BuildError: The custom build system failed.
Traceback (most recent call last):
  File "c:\python37_64\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python37_64\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\davidlatwe.lai\.rezup\dev\revisions\1637232169.1712413\venv\rez\Scripts\rez\rez-deliver.exe\__main__.py", line 7, in <module>
  File "c:\users\davidlatwe.lai\pipeline\rez-deliver\src\deliver\rezplugins\command\deliver.py", line 21, in rez_cli
    return run("deliver")
  File "c:\users\davidlatwe.lai\pipeline\rez\src\rez\cli\_main.py", line 187, in run
    returncode = run_cmd()
  File "c:\users\davidlatwe.lai\pipeline\rez\src\rez\cli\_main.py", line 179, in run_cmd
    return func(opts, opts.parser, extra_arg_groups)
  File "c:\users\davidlatwe.lai\pipeline\rez-deliver\src\deliver\rezplugins\command\deliver.py", line 77, in command
    if cli.deploy_packages(opts.PKG, path, opts.dry_run, opts.yes):
  File "c:\users\davidlatwe.lai\pipeline\rez-deliver\src\deliver\cli.py", line 72, in deploy_packages
    installer.run()
  File "c:\users\davidlatwe.lai\pipeline\rez-deliver\src\deliver\install.py", line 17, in run
    for _ in self.run_iter():
  File "c:\users\davidlatwe.lai\pipeline\rez-deliver\src\deliver\install.py", line 35, in run_iter
    ver_tag=requested.ver_tag)
  File "c:\users\davidlatwe.lai\pipeline\rez-deliver\src\deliver\install.py", line 78, in _build
    self._run_command(cmd, cwd=src_dir, env=env)
  File "c:\users\davidlatwe.lai\pipeline\rez-deliver\src\deliver\install.py", line 84, in _run_command
    subprocess.check_call(cmd_args, **kwargs)
  File "c:\python37_64\lib\subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['C:\\Users\\davidlatwe.lai\\.rezup\\dev\\revisions\\1637232169.1712413\\venv\\rez\\Scripts\\python.exe', '-m', 'deliver.install', 'usd_maya-0.14.0-m2[0]', '--install', '--variants', '0']' returned non-zero exit status 1.
```

</details>

**Specs:**
 - OS & version [Windows 10]
 - Compiler & version [VS-2017]
 - Maya version [Maya 2020]
 - Maya USD commit SHA [dev at 75e7112]
 - Pixar USD commit SHA [21.08 at [7a5f8c4](https://github.com/PixarAnimationStudios/USD/commit/7a5f8c4311fed3ef2271d5e4b51025fb0f513730)]

**Additional context**
Using `MAYA_API_VERSION` fixes the issue.

--- a/lib/mayaUsd/fileio/translators/translatorCurves.cpp
+++ b/lib/mayaUsd/fileio/translators/translatorCurves.cpp
@@ -61,7 +61,7 @@ bool convertToBezier(MFnNurbsCurve& nurbsCurveFn, MObject& mayaNodeTransformObj,
     // Remove the nurbs and converter:
     MDGModifier dagm;
     dagm.deleteNode(convFn.object());
-#if MAYA_APP_VERSION >= 2020
+#if MAYA_API_VERSION >= 20200300
     dagm.deleteNode(nurbsCurveFn.object(), false);
 #else
     dagm.deleteNode(nurbsCurveFn.object());

